### PR TITLE
Fill/Stroke with color should not crash, if color is null

### PR DESCRIPTION
**Pull request by msft-Jeyaram**

### CODE REVIEW

File: tests/UnitTests/CoreGraphics.drawing/CGContextDrawingTests.cpp [hunk 1]
```diff
@@ -213,4 +213,21 @@ DRAW_TEST_F(CGContext, AntialiasToggleTranscendsGState, WhiteBackgroundTest<>) {
         { 15, 5 }, { bounds.size.width, bounds.size.height - 10 },
     };
     CGContextStrokeLineSegments(context, blurryLineSegments2, 2);
-}
\ No newline at end of file
+}
+
+DRAW_TEST_F(CGContext, NullFillColor, WhiteBackgroundTest<>) {
```
Comment (msft-Jeyaram) [inline tests/UnitTests/CoreGraphics.drawing/CGContextDrawingTests.cpp]: hmm not sure if we need the drawing test.
Thoughts?

Comment (aballway) [inline tests/UnitTests/CoreGraphics.drawing/CGContextDrawingTests.cpp]: We only need to test that the fill/stroke colors aren't changed when nullptr is passed in correct? Is there a way to get the color from the context to check?

**Comment by DHowett-MSFT** [comment]
Does this hold on the reference platform? I have a post-it on my desk saying that we should support `NULL` colors; that implies to me that it's slightly more complicated than ignoring them.

**Comment by msft-Jeyaram** [comment]
@DHowett-MSFT  this does hold on reference platform.

File: Frameworks/CoreGraphics/CGContext.mm [hunk 2]
```diff
@@ -1650,6 +1650,8 @@ void CGContextSetLineWidth(CGContextRef context, CGFloat width) {
 */
 void CGContextSetStrokeColor(CGContextRef context, const CGFloat* components) {
     NOISY_RETURN_IF_NULL(context);
+    NOISY_RETURN_IF_NULL(components);
```
Comment (msft-Jeyaram) [inline Frameworks/CoreGraphics/CGContext.mm]: for these it uses the existing value.
It's covered in the test.

**Pull request merged**